src/components/safe-apps/AppFrame/TransactionQueueBar/index.tsx

```diff
@@ -0,0 +1,100 @@
+import type { Dispatch, ReactElement, SetStateAction } from 'react'
+import { Backdrop, Typography, Box, IconButton, Accordion, AccordionDetails, AccordionSummary } from '@mui/material'
+import { ClickAwayListener } from '@mui/base'
+import CloseIcon from '@mui/icons-material/Close'
+import ExpandLessIcon from '@mui/icons-material/ExpandLess'
+
+import useTxQueue from '@/hooks/useTxQueue'
+import PaginatedTxns from '@/components/common/PaginatedTxns'
+import styles from './styles.module.css'
+import { getQueuedTransactionCount } from '@/utils/transactions'
+// import { BatchExecuteHoverProvider } from '@/components/transactions/BatchExecuteButton/BatchExecuteHoverProvider'
+// import BatchExecuteButton from '@/components/transactions/BatchExecuteButton'
```

Comment:
I wasn't sure if these were removed for a specific reason. Currently the tx builder app fails, and I suspect it's because of these being missing.